site/app/templates/admin/SqlToolbox.twig

```diff
@@ -4,6 +4,28 @@
     <div>
       Use this toolbox to run a SELECT query. You cannot run any other type of query, and may only run a single query at a time.
       <br /><br />
+
+        <div>
+            <button id="sqlDatabaseSchema" class="btn btn-primary">Database Schema Documentation</button>
```

Comment:
Per our docs (https://submitty.org/developer/coding_style_guide/css), IDs and classes should be `kebab-case`, e.g. `sql-dashboard-schema`.